flag/service/service.go

```diff
@@ -11,6 +11,8 @@ type Service struct {
 	WriteAllCustomerGroup   string
 	WriteAllGiantswarmGroup string
 
+	ReadAllCustomerGroup string
```

Comment:
What's this for? I don't think this is necessary unless you're also adding a flag for a singular group name, which I also don't think is needed.

Reply:
I was planning to, but I don't think it's necessary